spyder/widgets/tests/helpers.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Helpers used in widget tests
+"""
+
+
+SHORTCUTS = {
+    'Close pane': ('Shift+Ctrl+F4' ,'⇧⌘F4'),
```

Comment:
I think you will need to use unicode strings for your tests to pass in Python 2.7, like `u'⇧⌘F4'`